src/pages/workspace/accounting/qbo/import/QuickbooksChartOfAccountsPage.tsx

```diff
@@ -35,9 +36,29 @@ function QuickbooksChartOfAccountsPage({policy}: WithPolicyProps) {
                 <HeaderWithBackButton title={translate('workspace.accounting.accounts')} />
```

Comment:
We have `ConnectionLayout` component which cleans up a lot of this header, title and access related to code. Should we use that?

Reply:
Interesting, I'll check, I didn't know about this component

Reply:
I used the new component `ConnectionLayout`. Thanks for the suggestion.